docs/src/administration/users.md

```diff
@@ -72,67 +60,27 @@ Available commands:
 
 ```bash
-platform user:add alice@example.com -r admin
+platform user:add user1@example.com -r admin
 ```
+Once you add the user to the project, they receive an invitation email to confirm their details and a registration link.
+
+**Example**
 
-After adding `alice@example.com`, Alice will receive an invitation email asking to confirm her details and optionally, register for a Platform.sh account.
+To give User2 different levels of access depending on the environment type in the current project:
 
-To give Bob different levels of access depending on the environment type, in the current project:
- 
 -  **Viewer** role to the **Production environment**
 -  **Contributor** role to **all Staging type environments**
 -  **Admin** role to **all Development type environments**
 
 You would run:
 
 ```bash
-platform user:role bob@example.com -r production:viewer -r staging:contributor -r development:admin
+platform user:role user2@example.com -r production:viewer -r staging:contributor -r development:admin
 ```
 
 Use `platform list` to get the full list of commands.
-
-## User access and integrations
-
-If you have setup an [external integration](/integrations/source/_index.md) to GitHub, GitLab, or Bitbucket, this adds an additional layer of access control to the project that you will need to be aware of.
-
-For example, if you invite a user with **Project Admin** role to a project on Platform.sh, but you haven't invited them to the remote repository on GitHub / GitLab, they might be [unable to clone the project](/administration/web/_index.md#git) locally.
-
-In this example, using either `platform get` with the CLI:
-
-```bash
-$ platform get <projectID>
-
-```
-
-or the `git clone` command visible from the "Git" dropdown in the management console
-
-```bash
-$ git clone git@github.com:user/github-repo.git Project Name
-```
-
-both would error with
-
-```bash
-Failed to connect to the Git repository: git@github.com:user/github-repo.git
-
-Please make sure you have the correct access rights and the repository exists.
-```
-
-despite their **Project Admin** access to the project.
-
-This enhaces consistency over your source code, as the Platform.sh project functions as a read-only mirror of your remote repository. Otherwise, changes pushed directly to the project would be overwritten or deleted when commits are pushed via the integration. Platform.sh considers your integrated remote repository to be the "source of truth" as soon as it has been configured, and this caveat ensures that all commits go through the integration.
-
-The best course of action is to have your access updated on the integrated repository. If for some reason that is not a quick change, you can still clone through the project using the legacy pattern (which will set the *project* as its remote), but again, it is not recommended that you commit to the project once you have done so:
-
-```bash
-$ git clone <project>@git.<region>.platform.sh:<project>.git
-```
```

Comment:
This information no longer exists anywhere. If you're planning on adding it, I'd add a cross link here to that information.

Reply:
I've got a question here: To me this piece of troubleshooting isn't really a part of this text and since it's new location is contextualized I wonder whether it is necessary to add a link from this page.

Reply:
Is there a reason you think it's not connected to this idea? If I were wondering why my user couldn't access something, I would look at a page called "User administration" and expect to find at least a link to the answer. But maybe that's just me :shrug:

Reply:
Ok